triagebot.toml

```
@@ -659,6 +659,7 @@ message_on_add = [
     """\
 /poll Approve stable backport of #{number}?
 approve
+approve (but does not justify new dot release)
```

Nit: maybe

```suggestion
approve (but does not justify new dot release on its own)
```

As in, `$team` would like to include it into a point release if there's other stable backport accepted candidates.